src/neo/Wallets/Wallet.cs

```diff
@@ -298,6 +301,32 @@ public Transaction MakeTransaction(TransferOutput[] outputs, UInt160 from = null
                                 }
                                 sb.EmitAppCall(output.AssetId, "transfer", account, output.ScriptHash, value);
                                 sb.Emit(OpCode.ASSERT);
+
+                                // Solid transfer
+
+                                if (output.SolidTransfer)
+                                {
+                                    byte[] solidTransferScript;
+                                    using (ScriptBuilder sbSolid = new ScriptBuilder())
+                                    {
+                                        sbSolid.EmitPush(value);
+                                        sbSolid.EmitAppCall(output.AssetId, "balanceOf", account);
+                                        sbSolid.Emit(OpCode.EQUAL);
```

Comment:
Should we default this to .EQUAL or to .GEQ?
Maybe, .GEQ is better, because parallel transfers would happen on Neo Wallet "by default". I think that, other user wallets (maybe Neon, etc), could focus on .EQUAL standard here, because use case is very targetted for non-automatic transfers (which is not true for Neo, as the "core Neo library").